src/commands/cmd_zset.cc

```diff
@@ -555,9 +561,17 @@ class CommandZRangeGeneric : public Commander {
         s = zset_db.RangeByRank(key_, rank_spec_, &member_scores, nullptr);
         break;
       case kZRangeScore:
+        if (score_spec_.count == 0) {
+          *output = redis::EmptyArray();
+          return Status::OK();
+        }
         s = zset_db.RangeByScore(key_, score_spec_, &member_scores, nullptr);
         break;
       case kZRangeLex:
+        if (lex_spec_.count == 0) {
+          *output = redis::EmptyArray();
```

Comment:
we can use `std::vector<std::string> values; *output =MultiBulkString(values, false)` to do the same thing.
I'm not sure if I want to add the specific `EmptyArray`.

Reply:
Yes, can remove it. Others are good to me.

Reply:
The third option is just `*output = MultiBulkString({});` since the array is empty and there is no need to care whether use an empty string or nil.
Other changes are good to me as well.